chainer/training/trigger.py

```diff
@@ -0,0 +1,69 @@
+class IntervalTrigger(object):
+
+    """Trigger based on a fixed interval.
+
+    This trigger accepts iterations divided by a given interval. There are two
+    ways to specify the interval: per iterations and epochs. `Iteration` means
+    the number of updates, while `epoch` means the number of sweeps over the
+    training dataset. Both values are defined by the updater.
+
+    For the description of triggers, see :func:`get_trigger`.
+
+    Args:
+        period (int): Length of the interval.
+        unit (str): Unit of the length specified by ``period``. It must be
+            either ``'iteration'`` or ``'epoch'``.
+
+    """
+    def __init__(self, period, unit):
+        self.period = period
+        assert unit == 'epoch' or unit == 'iteration'
+        self.unit = unit
+
+    def __call__(self, trainer):
+        """Decides whether the extension should be called on this iteration.
+
+        Args:
+            trainer (Trainer): Trainer object that currently runs.
```

Comment:
Consider rewording: The Trainer object that this trigger is associated with.

Is it worth mentioning that the trigger performs some read-only operations on the updater associated with the trainer to determine if the trigger should fire?